kube_dns/README.md

```diff
@@ -0,0 +1,60 @@
+# Kube-dns Integration
+
+## Overview
+
+Get metrics from kube-dns service in real time to:
+
+* Visualize and monitor dns metrics collected via Kubernetes' kube-dns addon
+  through Prometheus
+
+See https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/dns for
+more informations about kube-dns
+
+## Installation
+
+Install the `dd-check-kube_dns` package manually or with your favorite configuration manager
+
+## Configuration
+
+Edit the `kube_dns.yaml` file to point to your server and port, set the masters to monitor
+
+### Using with service discovery
```

Comment:
added the auto config.
I also put `%%port_2%%` in there but the problem is that it might not always be port_2 if you want to disable the tcp listening for the dns for example.
Config example on the pod:
```                 "image": "gcr.io/google_containers/kubedns-amd64:1.9",                                                                                                                                             
                "imagePullPolicy": "IfNotPresent",                                                                                                                                                                 
                "livenessProbe": {                                                                                                                                                                                 
                    "failureThreshold": 5,                                                                                                                                                                         
                    "httpGet": {                                                                                                                                                                                   
                        "path": "/healthz-kubedns",                                                                                                                                                                
                        "port": 8080,                                                                                                                                                                              
                        "scheme": "HTTP"                                                                                                                                                                           
                    },                                                                                                                                                                                             
                    "initialDelaySeconds": 60,                                                                                                                                                                     
                    "periodSeconds": 10,                                                                                                                                                                           
                    "successThreshold": 1,                                                                                                                                                                         
                    "timeoutSeconds": 5                                                                                                                                                                            
                },                                                                                                                                                                                                 
                "name": "kubedns",                                                                                                                                                                                 
                "ports": [                                                                                                                                                                                         
                    {                                                                                                                                                                                              
                        "containerPort": 10053,                                                                                                                                                                    
                        "name": "dns-local",                                                                                                                                                                       
                        "protocol": "UDP"                                                                                                                                                                          
                    },                                                                                                                                                                                             
                    {                                                                                                                                                                                              
                        "containerPort": 10053,                                                                                                                                                                    
                        "name": "dns-tcp-local",                                                                                                                                                                   
                        "protocol": "TCP"                                                                                                                                                                          
                    },                                                                                                                                                                                             
                    {                                                                                                                                                                                              
                        "containerPort": 10055,                                                                                                                                                                    
                        "name": "metrics",                                                                                                                                                                         
                        "protocol": "TCP"                                                                                                                                                                          
                    }                                                                                                                                                                                              
                ],  ```